centreon/www/front_src/src/AdditionalConnectorConfiguration/Listing/ActionsBar/Filters/useUpdateFiltersBasedOnSearchBar.ts

```diff
@@ -0,0 +1,62 @@
+import { useEffect } from 'react';
+
+import { find, includes, isEmpty, last, pipe, replace, trim } from 'ramda';
+import { useAtomValue, useSetAtom } from 'jotai';
+
+import { filtersAtom, searchAtom } from '../../atom';
+
+const excludeSubstring = (substring) => (str) => replace(substring, '', str);
+
+const useUpdateFiltersBasedOnSearchBar = (): void => {
```

Comment:
Add some memoization on functions and values as it does a lot of computation on a single render